jdaviz/app.py

```diff
@@ -643,6 +646,16 @@ def _get_all_subregions(mask, spec_axis_data):
                     regions[key] = subregions_in_subset
                     continue
 
+                temp_data = self.get_data_from_viewer(viewer_reference, value.label)
+                if isinstance(temp_data, Spectrum1D):
+                    # Note that we look for mask == False here, rather than True above,
+                    # because specutils masks are the reverse of Glue (of course)
```

Comment:
FWIW, `specutils` at least follows Numpy masked array convention (where `mask==True` means the pixel is masked). 🤷